src/OpenTelemetry.Instrumentation.Owin/Implementation/DiagnosticsMiddleware.cs

```diff
@@ -84,39 +86,26 @@ private static void BeginRequest(IOwinContext owinContext)
         {
             var request = owinContext.Request;
 
-            /*
-             * Note: Display name is intentionally set to a low cardinality
-             * value because OWIN does not expose any kind of
-             * route/template. See:
-             * https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name
-             */
-            activity.DisplayName = request.Method switch
-            {
-                "GET" => "HTTP GET",
-                "POST" => "HTTP POST",
-                "PUT" => "HTTP PUT",
-                "DELETE" => "HTTP DELETE",
-                _ => $"HTTP {request.Method}",
-            };
+            // Note: Display name is intentionally set to a low cardinality
+            // value because OWIN does not expose any kind of
+            // route/template. See:
+            // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name
+            RequestDataHelper.SetActivityDisplayName(activity, activity.DisplayName);
```

Comment:
I think that the name should be set based on the request.Method.

```suggestion
            RequestDataHelper.SetActivityDisplayName(activity, request.Method);
```